africanus/gridding/perleypolyhedron/policies/stokes_conversion_policies.py

```diff
@@ -5,7 +5,7 @@ def stokes2corr(vis_in, vis_out, policy_type):
     pass
 
 
-@overload(stokes2corr, inline="always")
+@overload(stokes2corr, inline="always", prefer_literal=True)
```

Comment:
@bennahugo Just letting you know that the test cases you wrote easily highlighted up the above needed change, due to the following https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-old-style-numba-captured-errors.

So the test cases were useful to have and I didn't have to understand the implementation to do the upgrade.